config/sync/ds.field.basic_page_type_content_url.yml

```diff
@@ -0,0 +1,11 @@
+id: basic_page_type_content_url
+label: 'Basic Page Type Content URL'
+ui_limit: 'stanford_page|*'
+properties:
+  content:
+    value: "<p>[node:url]</p>\r\n"
+    format: stanford_minimal_html
```

Comment:
I hate how `plain_text` doesn't work with the tokens. We should fix later that so these ds fields are cleaner.

Reply:
this is for a later date. just making note